src/python/WMCore/WMSpec/Steps/Executors/StageOut.py

```diff
@@ -136,48 +135,40 @@ def execute(self, emulator = None):
             # for that step; or so I hope
             files = stepReport.getAllFileRefsFromStep(step = step)
             for fileName in files:
-                if not hasattr(fileName, 'lfn') and hasattr(fileName, 'pfn'):
-                    # Then we're truly hosed on this file; ignore it
-                    msg = "Not a file: %s" % fileName
+
+                # make sure the file information is consistent
+                if hasattr(fileName, 'pfn') and ( not hasattr(fileName, 'lfn') or not hasattr(fileName, 'module_label') ):
```

Comment:
I don't understand this check either. I just kept the original check and added one more for module_label since I am actually using it, so it should be there. Based on the original check it almost looks like we don't want files for stageout that have a pfn and no lfn. But files with no pfn and no lfn are ok ???